easybuild/easyconfigs/u/UFL/UFL-2024.2.0-gfbf-2023b.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'PythonPackage'
+
+name = 'UFL'
+version = '2024.2.0'
+
+homepage = 'https://fenics.readthedocs.io/projects/ufl'
+
+description = """
+The Unified Form Language (UFL) is a domain specific language for
+declaration of finite element discretizations of variational forms.
+"""
+
+toolchain = {'name': 'gfbf', 'version': '2023b'}
+
+source_urls = ['https://github.com/FEniCS/ufl/archive/refs/tags/']
+sources = ['%(version)s.tar.gz']
+checksums = ['d9353d23ccbdd9887f8d6edab74c04fe06d818da972072081dbf0c25bc86f5a7']
+
+builddependencies = [
+    ('CMake', '3.27.6'),
+    ('scikit-build-core', '0.10.7'),
+]
+
+dependencies = [
+    ('Python', '3.11.5'),
+    ('SciPy-bundle', '2023.11'),
+    ('pybind11', '2.11.1'),
+]
+
+use_pip = True
+sanity_pip_check  = True
```

Comment:
```suggestion
sanity_pip_check = True
```

Reply:
The other CI failure (for checksums) will be fixed by https://github.com/easybuilders/easybuild-framework/pull/4806